src/protocol.cpp

```diff
@@ -2385,14 +2512,15 @@ bool CProtocol::EvaluateCLRedServerListMes ( const CHostAddress&     InetAddr,
         }
 
         // add server information to vector
-        vecServerInfo.Add (
-            CServerInfo ( CHostAddress ( QHostAddress ( iIpAddr ), iPort ),
-                          CHostAddress ( QHostAddress ( iIpAddr ), iPort ),
-                          strName,
-                          QLocale::AnyCountry, // set to any country since the information is not transmitted
-                          "", // empty city name since the information is not transmitted
-                          0, // per definition: if max. num. client is zero, we ignore the value in the server list
-                          false ) ); // assume not permanent since the information is not transmitted
+        vecServerInfo.Add ( CServerInfo (
+            CHostAddress ( QHostAddress ( iIpAddr ), iPort ),
+            CHostAddress ( QHostAddress ( iIpAddr ), iPort ),
+            strName,
+            QLocale::
+                AnyCountry, // set to any country since the information is not transmitted
```

Comment:
Agreed, that style break shouldn't be allowed.  Any `ClassName::AMemberName` entry should be treated as an inseparable element (along with leading / trailing `&`, if present, too).

Reply:
fine with `ColumnLimit: 120`

Reply:
Cool! Is there any generic way to avoid such breaks? I'm thinking of some kind of cohesiveness rules, e.g. "keep class name + constant together" should win over "keep inline-comment on the same line".

Reply:
The comment doesn't get broken here as there's `ReflowComments: false` in the style config.
I had disabled that as with `ColumnLimit: 80` it would have broken a very big number of comments.
with the higher ColumnLimit it should be ok to keep `ReflowComments` enabled